src/js/utils/patient-template.js

```diff
@@ -27,11 +27,11 @@ function escapeChar(match) {
 function deepGetTemplate(dataKey, nestedKeys) {
   let keys = '';
 
-  _.each(nestedKeys.split('.'), key => {
+  each(nestedKeys.split('.'), key => {
     keys += `'${ key }',`;
   });
 
-  return `'+\n((__t=(_.propertyOf(data.${ dataKey })([${ keys }])))==null?'':_.escape(__t))+\n'`;
+  return `'+\n((__t=(_.propertyOf(data.${ dataKey })([${ keys }])))==null?'':escape(__t))+\n'`;
```

Comment:
this looks problematic to me.. either `propertyOf` will work without underscore just importing it in or escape won't work.  If this is a linter issue we can add a comment to disable that import error.  I kinda think `_` isn't necessary here, but what we could do if this break on compile is something like:
```
import { propertyOf, escape, each, first, reduce } from underscore;
export const _ = { propertyOf, escape };
Then that `_` should be available to the template string.
```

Reply:
The only issue was that our linter complains that `propertyOf` is defined but never used because it's wrapped in the string. Thoughts on disabling eslint for the import line?